migrations/founders.kovan.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
do we need kovan as part of the file name ?

Reply:
there is also one with ganache in the name

Reply:
why not have just founders.json (for both ganache and kovan ).

Reply:
for all networks